source/whats-new.txt

```diff
@@ -0,0 +1,34 @@
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
```

Comment:
Issue:

Style guide on lists: "Avoid using fragments to introduce lists. Fragments can be harder to understand than sentences."

Suggestion:
Convert the fragment into a sentence by identifying an object: "Learn what's new in the following versions:"

Reply:
I totally agree. Unfortunately, I was using our other What's New pages as a guide, so those need updating too!